tests/libsubid/04_nss/libsubid_zzz.c

```diff
@@ -76,7 +76,7 @@ static uid_t getnamuid(const char *name) {
 }
 
 static int alloc_uid(uid_t **uids, uid_t id) {
-	*uids = malloc(sizeof(uid_t));
+	*uids = MALLOC(uid_t);
```

Comment:
Oh, this is actually breaking build of libsubid_zzz during tests.

Not sure yet whether just adding #include "alloc.h" at the top suffices.

Reply:
Yes, that should work.

Reply:
I'm not sure if that library links to libmisc, though, which might be a problem if not.